app/temperature.py

```diff
@@ -0,0 +1,26 @@
+import httpx
+from sqlalchemy.orm import Session
+from datetime import datetime
+from app import models, schemas
+
+
+async def fetch_temperature(city_name: str):
+    response = await httpx.get(f"http://api.openweathermap.org/data/2.5/weather?q={city_name}&appid=YOUR_API_KEY")
```

Comment:
The API key is hardcoded as 'YOUR_API_KEY'. Replace this with a valid API key or use environment variables to manage sensitive information securely.